### PR TITLE
remove package pflag in pd-ctl

### PR DESCRIPTION
**Pull request by loxp**
### What problem does this PR solve? 
UCP #1832 

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

Side effects

Related changes

### CODE REVIEW

File: tools/pd-ctl/pdctl/ctl.go [hunk 1]
```diff
@@ -14,10 +14,14 @@
 package pdctl
 
 import (
-	"os"
-
+	"fmt"
```
Comment (nolouch) [inline tools/pd-ctl/pdctl/ctl.go]: Suggest import as group ordering, see[ more](https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering).

File: tools/pd-ctl/pdctl/ctl.go [hunk 2]
```diff
@@ -68,11 +87,55 @@ func Start(args []string) {
...
+
+	rootCmd.SetArgs(args)
 	rootCmd.ParseFlags(args)
 	rootCmd.SetOutput(os.Stdout)
 
+	return rootCmd
+}
+
+// MainStart start main command
+func MainStart(args []string) {
```
Comment (rleungx) [inline tools/pd-ctl/pdctl/ctl.go]: Looks like these two functions are almost the same except the `rootCmd`, can we combine them?

File: tools/pd-ctl/pdctl/ctl.go [hunk 3]
```diff
@@ -31,23 +35,38 @@ type CommandFlags struct {
...
+	}
+}
+
+func getBasicCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:   "pd-ctl",
 		Short: "Placement Driver control",
 	}
+
 	rootCmd.PersistentFlags().StringVarP(&commandFlags.URL, "pd", "u", "http://127.0.0.1:2379", "pd address")
```
Comment (rleungx) [inline tools/pd-ctl/pdctl/ctl.go]: It's better to change all the first letters to the upper case.

Comment (loxp) [reply tools/pd-ctl/pdctl/ctl.go]: Do you mean change all first letters of functions in ctl.go to upper case?

Comment (rleungx) [reply tools/pd-ctl/pdctl/ctl.go]: oh, sorry for misleading. What I mean is the help message.

File: tools/pd-ctl/pdctl/ctl.go (hunk 2 above)
Comment (loxp) [reply tools/pd-ctl/pdctl/ctl.go]: I refactor the code, PTAL.

File: tools/pd-ctl/pdctl/ctl.go (hunk 3 above)
Comment (loxp) [reply tools/pd-ctl/pdctl/ctl.go]: In fact, only the MainStart() needs to export to package main.

**Comment by loxp** [comment]
@rleungx BTW, I think the `detach` flag is unused in pd-ctl. Can we remove it?

File: tools/pd-ctl/pdctl/ctl.go (hunk 3 above)
Comment (loxp) [reply tools/pd-ctl/pdctl/ctl.go]: > It's better to change all the first letters to the upper case.

OK, change the first letter in message to upper case now.

**Comment by nolouch** [comment]
@loxp   `detach` flag is reserved for compatibility, previously `pd-ctl` default mode was interactive.

**Pull request merged**